bridge/svix-bridge-plugin-queue/tests/sqs_consumer.rs

```diff
@@ -74,7 +84,7 @@ async fn publish(client: &Client, url: &str, payload: &str) {
 }
 
 /// General "pause while we wait for messages to travel" beat. If you're seeing flakes, bump this up.
-const WAIT_MS: u64 = 100;
+const WAIT_MS: u64 = 300;
```

Comment:
Not sure what accounts for this, but SQS got a lot more flaky with omniqueue. Might be able to reduce this again once "de-bulking" has happened. Fewer trait objects in the call graph might make a difference.

Reply:
This is very concerning, no? As in, it feels like something is bad with omniqueue and we shouldn't release bridge with this?

Reply:
I'm not quite so concerned right now. I've been more worried about correctness during this refactor process 😅 

I'll try and step it back down now that the diff has settled.

There's a range in use across the different backends:
- gcp, 100
- rabbit, 200
- redis, 250

tbh, it's hard to say if this was related to resource starvation on my machine while building/running. SQS was the only one I saw flake though.

Reply:
OK, just feels like it's worth investigating.

Reply:
Managed to pull it down to 250 so at least it's in line with redis 🤣 

I think it's probably okay. We can definitely do a separate pass to tighten things up on both sides, here and omniqueue.

Reply:
Filed https://github.com/svix/monorepo-private/issues/6140 to follow-up on this.

Reply:
Filed an issue to look into this further.